src/main/datatypes/MQ2AdvLootType.cpp

```diff
@@ -345,7 +347,49 @@ bool MQ2AdvLootItemType::GetMember(MQVarPtr VarPtr, const char* Member, char* In
 		Dest.Type = pBoolType;
 		return true;
 
-	default: break;
+	case AdvLootItemMembers::Status:
+	{
+		Dest.Type = pStringType;
+
+		std::string tmp_str = "INVALID";
+
+		// Check to make sure we aren't trying to access the personal loot window and if so return INVALID.
+		if( itemList == pAdvancedLootWnd->pPLootList )
```

Comment:
Is there a reason you are formatting like this? This is inconsistent with the rest of the code base -- it should be `if (thing)`